.gitignore

```diff
@@ -1,2 +1,6 @@
 Cargo.lock
 target/
+
+*/*.log
+*/crash-*
+*/corpus
```

Comment:
Just saw #7. I added `seeds` directories for user supplied data but seeing as the fuzzer generated corpus can become quite large (lots of tiny files) I didn't want to add that here.